src/commands/startCCFNetworkInDocker.ts

```diff
@@ -58,7 +64,10 @@ export async function startCCFNetworkDocker() {
     // such that we will execute the subsequent commands only if the previous ones succeed
     const commandsSequence = [dockerBuildCommand, dockerRunCommand];
 
-    const finalCommand = commandsSequence.join(" && ");
+    const finalCommand =
+      envType == TerminalType.PowerShell
+        ? commandsSequence.join(" ; ")
```

Comment:
Shouldn't we use `-and` instead of `;` ? That's the equivalent to using `&&` in linux / cmd

Reply:
No, for "-and" it's only used for `logical` operations within PowerShell command. So if you use this in a regular command, it will not be recognized. The correct way to use it is with ";" in powershell.

Reply:
![image](https://github.com/microsoft/mccf-vscode-extension/assets/111912464/bccc1a45-b3c3-4717-88d1-b7b774aa71e6)
FYI, like this the flag won't be recognized.